jep/0000/README.adoc

```diff
@@ -0,0 +1,316 @@
+* 2.307-centos7
+* 2.307-slim
+* alpine
+* centos7
+* latest
+* slim
+
+Change to JDK 11 in the link:https://hub.docker.com/r/jenkins/jenkins[long term support controller images] that do not include an explicit JDK in their tag, like:
+
+* 2.302.1
+* 2.302.1-alpine
+* 2.302.1-centos7
+* 2.302.1-slim
+* lts
+* lts-alpine
+* lts-centos7
+* lts-slim
+
+Change to JDK 11 in the link:https://hub.docker.com/r/jenkins/agent/[agent images] that do not include an explicit JDK in their tag, like:
+
+* 4.9-1
+* 4.9-1-alpine
+* 4.9-1-windowsservercore-ltsc2019
+* alpine
+* archlinux
+* latest
+* latest-archlinux
+* stretch - stop publishing updates to this image, replace with buster image
+
+Change to JDK 11 in the link:https://hub.docker.com/r/jenkins/inbound-agent[inbound agent images] that do not include an explicit JDK in their tag, like:
+
+* 4.9-1
+* 4.9-1-alpine
+* 4.9-1-windowsservercore-ltsc2019
+* alpine
+* latest
+
+Change to JDK 11 in the link:https://hub.docker.com/r/jenkins/ssh-agent/[outbound (ssh) agent images] that do not include an explicit JDK in their tag, like:
+
+* 2.1.0-alpine
+* 3.0.0
+* 3.0.0-alpine
+* alpine
+* latest
+* stretch - no change, retire the image
+
+=== Add new tags and images
+
+Additional tags will be added for those users who require a Java 8 image.
+
+**Provide additional Docker images for Java 8** in case users have critical requirements that must use Java 8.
+
+Add JDK 8 tags for the link:https://hub.docker.com/r/jenkins/jenkins[weekly controller images] as a fallback for users requiring JDK 8:
+
+* latest-jdk8
+* slim-jdk8
+* centos7-jdk8
+* alpine-jdk8
+
+Add JDK 8 tags for the link:https://hub.docker.com/r/jenkins/jenkins[long term support controller images] that do not include an explicit JDK in their tag, like:
+
+* lts-jdk8
+* lts-slim-jdk8
+* lts-centos7-jdk8
+* lts-alpine-jdk8
+
+Add JDK 8 tags to the link:https://hub.docker.com/r/jenkins/inbound-agent[inbound agent images] that do not include an explicit JDK in their tag, like:
+
+* 4.9-1-jdk8
+* 4.9-1-jdk8-alpine
+* 4.9-1-jdk8-windowsservercore-ltsc2019
+* alpine-jdk8
+* latest-jdk8
+
+Add JDK 8 tags to the link:https://hub.docker.com/r/jenkins/ssh-agent/[outbound (ssh) agent images] that do not include an explicit JDK in their tag, like:
+
+* 2.1.0-alpine-jdk8
+* 3.0.0-jdk8
+* 3.0.0-alpine-jdk8
+* alpine-jdk8
+* latest-jdk8
+* stretch no change, retire the image
+
+Some of the existing tags are for operating systems that have ended their standard support life.
+Add new operating system tags to use actively maintained operating systems.
+
+**Add Docker images for Debian 10 (buster)** in those images that include a link:https://www.debian.org/releases/stretch/[Debian 9 (stretch)] image.
+Debian 9 switched to link:https://wiki.debian.org/LTS[long term support] July 9, 2020.
+Debian 9 link:https://wiki.debian.org/LTS[long term support] will end June 30, 2022.
+
+Add Debian Buster image to the outbound (ssh) agent images to replace Debian stretch:
```

Comment:
Should we just go straight to bullseye? It’s switching to Stable on 14th august

Reply:
I am +1 though it is not related to JEP draft process.
Let's move to the mailing list?

Reply:
🤷 it's mentioned here as what we're going to do.

Reply:
I wasn't aware that Debian Bullseye had set a release date, but a [July 23, 2021 mailing list message](https://lists.debian.org/debian-devel-announce/2021/07/msg00003.html) confirms Aug 14, 2021 is their intended date.  Yes, we should go to bullseye, though if they slip, it may complicate this process.

Reply:
Included that change in 1da5ad62fd28046548e6b384bcabde427d379506